ssm/ssm/projects/migrations/0007_membershipmodel_hours_per_day.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.5 on 2019-02-15 08:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0006_auto_20190215_0705'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='membershipmodel',
+            name='hours_per_day',
+            field=models.IntegerField(default=8, verbose_name='Hours per day'),
+        ),
+    ]
```

Comment:
Please try to use less amount of migrations before making your pull request, in your case when you do one new app support from scratch you can always remove all migrations for project at the end and recreate them to have clear picture for reviewer without all these changes history. Of course you shouldn't use such deletion migration practice in all other cases, but for current workflow it's quite a good one